docs/creating-and-editing-specialist-document-types.md

```diff
@@ -1,20 +1,15 @@
 # Creating or editing specialist document types
 
 To create or edit a new specialist document you will have to make changes to this
-application, [govuk-content-schemas][govuk-content-schemas] and
-[search-api][search-api]. You will not have to make any changes to frontend
+application, [publishing-api][publishing-api] and [search-api][search-api]. You will not have to make any changes to frontend
 applications.
 
-[govuk-content-schemas]: https://github.com/alphagov/govuk-content-schemas
+[publishing-api]: https://github.com/alphagov/publishing-api
 [search-api]: https://github.com/alphagov/search-api
 
 # __Creating__ a specialist document type
 
-## 1. Add a schema to govuk-content-schemas
-
-See example [PR for adding `product_safety_alert`](https://github.com/alphagov/govuk-content-schemas/pull/1077).
-Note: this is a link to a PR in the now-archived content schemas repo. Content schemas have now been moved to publishing api.
-Please update this document when an example for publishing api is available.
+## 1. Add a schema to Publishing API
```

Comment:
Do we have an example link for this?

There's this one but I don't think it's the best, maybe would could explain it if there's no example?
like add the details in this format then run the build schemas rake task

Reply:
I could add my one? 😅 https://github.com/alphagov/publishing-api/pull/2589/files

Reply:
Looks good to me! Maybe for the next one we could separate the changes from the build (it's annoying to do but nice for an example), luckily we have loads of finders so I'm sure we can do one and update it if we need to!